src/sandbox/console/index.js

```diff
@@ -1,16 +1,18 @@
 import { dispatch } from 'codesandbox-api';
 
-import massageJSON from './massage-json';
+import messageJSON from './message-json';
 
```

Comment:
Haha, this wasn't a typo, but a bit cryptic. Maybe rename it to transform-json?

Reply:
Haha, **now** I get it!

Reply:
Renamed it to `transform-json.js` / `transformJSON`.

Reply:
Great! Yeah massage sounds a bit strange. Heard it last week at the conference haha